demos/python_demos/handwritten_japanese_recognition_demo/README.md

```diff
@@ -0,0 +1,82 @@
+
+# Handwritten Japanese Recognition Demo
+This example demonstrates an approach to recognize handwritten japanese text line using OpenVINO<sup>TM</sup>. This model supports all the characters in the [Kondate](http://web.tuat.ac.jp/~nakagawa/database/en/kondate_about.html) dataset.
+
+## How It Works
+
+The demo expects the next model in the Intermediate Representation (IR) format:
+
+   * handwritten-japanese-recognition-0001
+
+It can be your own models or pre-trained model from OpenVINO Open Model Zoo.
+In the `models.lst` are the list of appropriate models for this demo
+that can be obtained via `Model downloader`.
+Please see more information about `Model downloader` [here](../../../tools/downloader/README.md).
+
+
+The demo workflow is the following:
+The demo firstly reads an image and perform the preprocessing such as resize, normalization and padding. Then after loading model to the IEPlugin, the inference will start. After decoding the returned indexes into chars, the demo will display the predicted text.
+
+
+### Installation and dependencies
+
+The demo depends on:
+- OpenVINO library (2019R3.1 or newer)
+- Python (3.5+, which is supported by OpenVINO)
```

Comment:
OMZ is a part of the OpenVINO toolkit, so it's unnecessary to specify these dependencies.